Viper Practice/src/controllers/TasksViewController.swift

```diff
@@ -11,7 +11,7 @@ class TasksViewController: UIViewController {
 
     @IBOutlet weak var tableView: UITableView!
     var presenter : TaskPresenterProtocol?
-    var tasks = [Task]()
+    var tasks = [Fruit]()
```

Comment:
same renaming for this class as well.
Also, why are you holding data in the viewController?

Our presenter should hold the data

Reply:
create a private var fruites in presenter and add getFruites func of return type array of fruites in presenter protocol and then use it in view controller.
let me know if this approach is okay or i have to create object of presenter class in view controller to get fruites data?